src/tests/snapshots/Quote.test.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import Quote from '../../pages/Quote';
+
+describe('Quote', () => {
+  test('should match Quote snapshot', () => {
+    const tree = render(
+      <Quote />,
+    );
+    expect(tree).toMatchSnapshot();
+  });
+});
```

Comment:
- [ ] @team great work on your components test however you forgot to add tests for the App component, kindly add it.